src/test/java/org/cactoos/scalar/FirstOfTest.java

```diff
@@ -53,6 +53,24 @@ public void returnsMatchingValue() {
         );
     }
 
+    @Test
+    public void returnsMatchingValueWithExceptionalFallback() {
+        final int value = 2;
+        MatcherAssert.assertThat(
```

Comment:
@iakunin We're using `cactoos-matchers` `Assertion`  instead of `MatcherAssert.assertThat`

Reply:
@paulodamaso sure, it's done.